src/scss/includes/search_form.scss

```
@@ -86,7 +86,7 @@ input[type='search'] {
 .search_form__action {
   width: 24px;
   height: calc($input-height - calc(2 * var(--memphis-border-width)));
-  background: url(../../images/magnifier-dark.svg) center no-repeat;
+  background: url(../../images//regular_icons/icons-regular-icon-search.svg) center no-repeat;
```

I don't think this `//` is intentional?

```suggestion
  background: url(../../images/regular_icons/icons-regular-icon-search.svg) center no-repeat;
```